### PR TITLE
Various tweaks to the FAQ

### PR DESCRIPTION
**Pull request by qwertychouskie**
Two spaces after sentence ends, mention Wiimote support (the article only being accessible from "Get involved" was a bit weird), grammar fixes, etc.

### CODE REVIEW

**Comment by MTres19** [comment]
No. Two spaces at the end of sentences is incorrect. Proper English is a requirement for the site to retain its professionalism. Sorry.

**Comment by qwertychouskie** [comment]
Fixed.  Sorry about that, my mom taught me wrong :(

**Comment by MTres19** [comment]
No problem. Merging.

File: FAQ.mw [hunk 1]
```diff
@@ -81,12 +81,16 @@ In 2015, we released version 0.9, a groundbreaking release powered by a brand ne
 |answer=The network support feature is under development. The basic support has been added in the Google Summer of Code 2013, but it's still not finished. See [[Talk:Networked Multiplayer]] (and pages linked from there) for more information.
 }}
 {{faq-question
-|title=Why some keyboard keys do not work when pressed simultaneously?
+|title=Why do some keyboard keys do not work when pressed simultaneously?
```
Comment (konstin) [inline FAQ.mw]: The first do is definitely too much here.

Comment (MTres19) [inline FAQ.mw]: Should have caught that. That's embarrassing since I'm a native English
speaker.

Comment (qwertychouskie) [inline FAQ.mw]: Oops, should have been "Why *do* some keyboard *keys not work* when pressed simultaneously?"

**Pull request merged**